projects/unrar/build.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash -eu
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+set -eu
+
+tar xf $SRC/unrarsrc-5.5.8.tar.gz
+
+# build 'lib'. This builds libunrar.a and libunrar.so
+# -fPIC is required for successful compilation.
+make CXX=$CXX CXXFLAGS="$CXXFLAGS -fPIC" -C $SRC/unrar/unrar lib
+
+# remove the .so file so that the linker links unrar statically.
+rm -v $SRC/unrar/unrar/libunrar.so
+
+cat <<HERE > $SRC/unrar/unrar_fuzzer.cc
```

Comment:
I think it's better to put this in a file and use the `COPY` docker command to copy it into the container instead of catting it here.

Reply:
Done